dune-project

```diff
@@ -10,7 +10,6 @@
  (synopsis "Deploy other OCurrent pipelines and services")
  (depends
   (ocaml (>= 4.14))
-  ; Provided by git submodules
   (current (>= 0.6.4))
```

Comment:
0.6.4 seems much older than the submodule commit we had. Should we bump up the version to the latest one or 0.6.5 which seems closer to the version we had pinned?

Reply:
The latest version will be installed anyway...  I would only add a constraint if the project couldn't be built on the earlier version due to an API change or something.  If we update this each time a new version is released we've swapped one maintenance task for another.

Reply:
This makes sense to me.